test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ModelExpressionTagHelper_Runtime.codegen.cs

```diff
@@ -45,7 +45,7 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_ModelExpre
             WriteLiteral("\r\n");
             EndContext();
             BeginContext(66, 25, false);
-            __tagHelperExecutionContext = __tagHelperScopeManager.Begin("input-test", global::Microsoft.AspNetCore.Razor.TagHelpers.TagMode.SelfClosing, "test", async() => {
+            __tagHelperExecutionContext = __tagHelperScopeManager.Begin("input-test", global::Microsoft.AspNetCore.Razor.TagHelpers.TagMode.SelfClosing, "16b8f99693bceb6f66cb680aaae1d399", async() => {
```

Comment:
I would think that we would still want to suppress the unique Ids in our tests. 

Think about what happens. You change one character in the source code, and guids change all over the place.

Reply:
Yes that would be very annoying 👍